Source/Generic/ECTestCase.m

```diff
@@ -695,7 +695,8 @@ - (NSBitmapImageRep*)bitmapAs32BitRGBA:(NSBitmapImageRep*)bitmap
 	[NSGraphicsContext setCurrentContext:ctx];
 
 	NSRect rect = NSMakeRect(0, 0, width, height);
-	[bitmap drawInRect:rect fromRect:rect operation:NSCompositeCopy fraction:1.0 respectFlipped:NO hints:nil];
+	NSRect fromRect = NSMakeRect(0, 0, bitmap.size.width, bitmap.size.height);
+	[bitmap drawInRect:rect fromRect:fromRect operation:NSCompositeCopy fraction:1.0 respectFlipped:NO hints:nil];
```

Comment:
Rather than calculate `fromRect`, how about just passing in `NSZeroRect`, which is documented to draw the whole image?